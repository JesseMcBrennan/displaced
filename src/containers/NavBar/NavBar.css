.navBar {
  width: 97.2%;
  height: 4em;
  background-color: rgba(0, 0, 0, 0.5);
  margin: 0;
  padding-top: 1.1%;
}

ul {
  test-align: center;
  list-style: none;
  width: 100%;
}

h1 {
  margin: 0;
  padding-top: 1%;
  font-family: 'Reenie Beanie', cursive;
  Font Family: Reenie Beanie;
  Font-Style: Regular;
  Font-Size: 48px;
  Line-Height: 48px;
  Align: Left;
  display: inline;
  color: #C4C4C4;
}

.title-middle {
  color: #FFFFFF;
  text-decoration: underline;
}

.navLink {
  text-decoration: none;
}
