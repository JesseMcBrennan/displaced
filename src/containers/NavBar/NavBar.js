import React, { Component } from 'react';
import { connect } from 'react-redux';
import { withRouter, NavLink } from 'react-router-dom';
import './NavBar.css'

export const NavBar = () => {
  return(
    <ul className='navBar'>
      <li></li>
      <li></li>
      <li></li>
    </ul>
  )
}