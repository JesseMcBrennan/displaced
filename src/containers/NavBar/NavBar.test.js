import React from 'react';
import ReactDom from 'react-dom';
import { shallow, mount } from 'enzyme';
import { NavBar } from './NavBar'

describe('NavBar container', () => {
  it('should render without crashing', () => {
    
  })
})