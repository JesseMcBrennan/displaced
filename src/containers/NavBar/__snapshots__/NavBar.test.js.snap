// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar container should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="navLink"
          replace={false}
          to="/"
>
          <h1>
                    repre
          </h1>
          <h1
                    className="title-middle"
          >
                    sent
          </h1>
          <h1>
                    ed
          </h1>
</Link>,
        <li />,
        <li />,
        <li />,
      ],
      "className": "navBar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <h1>
              repre
</h1>,
            <h1
              className="title-middle"
>
              sent
</h1>,
            <h1>
              ed
</h1>,
          ],
          "className": "navLink",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "repre",
            },
            "ref": null,
            "rendered": "repre",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "sent",
              "className": "title-middle",
            },
            "ref": null,
            "rendered": "sent",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "ed",
            },
            "ref": null,
            "rendered": "ed",
            "type": "h1",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "li",
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="navLink"
            replace={false}
            to="/"
>
            <h1>
                        repre
            </h1>
            <h1
                        className="title-middle"
            >
                        sent
            </h1>
            <h1>
                        ed
            </h1>
</Link>,
          <li />,
          <li />,
          <li />,
        ],
        "className": "navBar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <h1>
                repre
</h1>,
              <h1
                className="title-middle"
>
                sent
</h1>,
              <h1>
                ed
</h1>,
            ],
            "className": "navLink",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "repre",
              },
              "ref": null,
              "rendered": "repre",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "sent",
                "className": "title-middle",
              },
              "ref": null,
              "rendered": "sent",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "ed",
              },
              "ref": null,
              "rendered": "ed",
              "type": "h1",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "li",
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
